app/Article.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Article extends Model
+{
+    protected $fillable = ['name', 'piece_start_stock', 'piece_min_stock', 'piece_max_stock', 'piece_order_stock', 'piece_weight', 'unit_start_stock', 'unit_min_stock', 'unit_max_stock', 'unit_order_stock', 'unit_weight', 'unit_size', 'location', 'location_maxweight', 'image'];
```

Comment:
With this many attributes you might wanna use $guarded instead of $fillable and only specify attributes that shall not be mass assignable. https://laravel.com/docs/7.x/eloquent#mass-assignment

Reply:
setting $guarded to an empty array seemed like the easy way out.